communitiesuk/funding-service-design-post-award-data-store
# FPASF-90: Fix broken local seed methods

### Change description
This PR addresses a recurring issue with the `flask seed` command where the hardcoded UUIDs for the `project_geospatial_association` table don't match those generated by the DB migrations that seed the `geospatial_dim` table.

It removes data migrations from past migration files (for `geospatial_dim` and `fund_dim`) that are now present in deployed environments and no longer needed. It also updates the CLI methods so `flask seed-ref` now seeds the same correct reference data and removes the individual reference data seeding methods and the dummy submission data seed method, which instead can be replicated by ingesting dummy spreadsheets locally.

This PR also fixes an inconsistency between the test csvs and the correct seeded data where the `geospatial_dim.itl1_region_name` value of `Yorkshire` was updated to `Yorkshire and The Humber`, but not updated in the csv. The `geospatial_dim` csv is also updated to use consistent UUIDs as `fund_dim` does.



### How to test
Drop the data in your database or rebuild your database container and bring up the services using docker compose. The database should be completely empty, with no data seeded by running migrations, and you can now run `flask seed-ref` to seed reference data. The docker compose start-up is being updated in [this separate PR](https://github.com/communitiesuk/funding-service-design-post-award-docker-runner/pull/36) to run `flask seed-ref` on start-up to remove this manual step.
